src/internal/util/reportError.ts

```diff
@@ -0,0 +1,34 @@
+import { Subscriber } from '../Subscriber';
```

Comment:
Since this is only used in one spot, you can just inline everything in that one spot. It'll perform better, and will be more readable (if a little harder to test)

Reply:
For tests, I'd rather just have more integration/functional tests of this.